kibana-reports/server/routes/utils/helpers.ts

```diff
@@ -20,11 +20,20 @@ import {
   ILegacyScopedClusterClient,
 } from '../../../../../src/core/server';
 
+// error response body:
```

Comment:
Should we leave this comment in?

Reply:
It serves as a reference of why we are doing the parsing below